main.py

```diff
@@ -8,10 +8,22 @@
 
 @app.route('/', methods=["POST", "GET"])
 def main():
-    #pull_request = PullRequest(request.get_json())
-    pull_request = PullRequest()
-    pull_request.set_issue_url('https://api.github.com/repos/dimagi/commcare-android/issues/1858')
-    return str(create_status_json(pull_request, REQUIRED_LABELS_ANY, REQUIRED_LABELS_ALL, BANNED_LABELS))
+    #event_json = request.get_json()
+    with open('./tests/pr_event_no_labels.json') as json_file:
```

Comment:
is this test code? not sure we should add that to the repo. you should be able to test the endpoint with curl and pass it in this test file if you wanted (or ideally, write a unit test that does what you want it to).